lib/plug/conn.ex

```diff
@@ -766,6 +767,7 @@ defmodule Plug.Conn do
     cond do
       opts[:renew] -> put_private(conn, :plug_session_info, :renew)
       opts[:drop]  -> put_private(conn, :plug_session_info, :drop)
+      opts[:ignore]  -> put_private(conn, :plug_session_info, nil)
```

Comment:
I would also store it as `:ignore`, even if it means duplicating two lines in the session plug. :)